org.flozz.calcleaner.yml

```diff
@@ -0,0 +1,31 @@
+app-id: org.flozz.calcleaner
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+command: calcleaner
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+modules:
+  - name: calcleaner
+    buildsystem: simple
+    build-commands:
+      # Build locales
+      - ./build-locales.py
```

Comment:
why isn't this part of upstream?

Reply:
On upstream repository this code is in a [nox session](https://github.com/flozz/calcleaner/blob/master/noxfile.py#L71-L88), but it were too complicated to have nox working in the build environment (too many dev dependencies...) so I preferred duplicating the code.

Reply:
why not keeping it upstream then?

(I have no idea what a nox session is)

Reply:
You can see Nox as « Makefile » specialized in running Python tests (it can setup specific Python version and install dependencies). I added the local extraction / update / compilation logic in it because it was convenient for me in my dev workflow